core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -101,13 +101,17 @@
                        angular-html-bind="activeCard.interactionHtml">
                   </div>
                 </div>
-                <div ng-if="!exploration.isInteractionInline(activeCard.stateName)"
-                     style="opacity: 0.8;">
-                  <md-button class="instructions-button"
-                          ng-click="showSupplementalCardIfScreenIsNarrow()">
+                <div ng-if="!exploration.isInteractionInline(activeCard.stateName)" style="opacity: 0.8;">
+                  <div ng-if="isCurrentSupplementalCardNonempty() && !isScreenNarrowAndShowingTutorCard()" style="text-transform: uppercase; padding: 6px 12px;"> 
                     <[exploration.getInteractionInstructions(activeCard.stateName)]>
                     <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
-                  </md-button>
+                  </div>
+                  <div ng-if="(!isCurrentSupplementalCardNonempty() || isScreenNarrowAndShowingTutorCard())">
+                    <md-button class="instructions-button" ng-click="showSupplementalCardIfScreenIsNarrow()">
+                      <[exploration.getInteractionInstructions(activeCard.stateName)]>
+                      <i ng-if="!isScreenNarrowAndShowingTutorCard()" class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
```

Comment:
I did this as a defensive measure as I'm not completely sure when the current supplemental card will be empty and the screen was wide enough. But. writing it down, I can see that this will never be needed to display. Have removed now.